docs/usage.rst

```diff
@@ -10,3 +10,16 @@ Run the ``CanarieAPI`` using ``gunicorn``::
     cd <CanarieAPI-root>/canarieapi
     ../bin/gunicorn -b 0.0.0.0:2000 --workers 1 --log-level=DEBUG --timeout 30 -k gevent wsgi
 
+
+Run the monitoring and/or the log parsing task as cron jobs.
+
+To run the log parsing job, add the following to a crontab file::
+
+    * * * * * python3 -c 'from canarieapi import logparser; logparser.cron_job()' 2>&1
+
+The log parsing job requires that a nginx reverse proxy service is running and the ``DATABASE["access_log"]``
+variable is set in the ``canarieapi/configuration.py`` script.
+
+To run the the monitoring job, add the following to a crontab file::
+
+    * * * * * python3 -c 'from canarieapi import monitoring; monitoring.cron_job()' 2>&1
```

Comment:
The commands differ from the sample cron definitions. 

Here, you could use `.. literalinclude:: canarieapi-cron` to avoid maintaining both places
(https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/code/literalinclude/literalinclude.html). Use `:lines:` to select each instruction individually.

Reply:
The example here would be for running cron on the host machine (not on docker) so the stdout/stderr redirects and the conditional execution isn't as necessary.

I'm assuming, based on the rest of this file, that this documentation is intended as "basic usage when running canarie-api locally" and is different from the docker usage. I didn't notice that there were docker specific usage documentation but if I've missed it let me know.

Reply:
No, there was no docker specific usage. 
Even this doc is outdated because `../bin/gunicorn` is not even in the repo anymore (just run `gunicorn` installed in the python env). 

It's ok to leave those examples as is. Personally, I debug calling the python commands directly instead of using cron. I don't expect many usages outside of Docker except for developers.